src/test/crimson/test_config.cc

```diff
@@ -1,26 +1,73 @@
 #include <chrono>
+#include <string>
 #include <numeric>
 #include <seastar/core/app-template.hh>
 #include <seastar/core/sharded.hh>
+#include "common/config_obs.h"
 #include "crimson/common/config_proxy.h"
 
 using Config = ceph::common::ConfigProxy;
+const std::string test_uint_option = "osd_max_pgls";
+
+class ConfigObs : public ceph::internal::md_config_obs_impl<Config> {
+  uint64_t last_change;
```

Comment:
please always initialize the member variables. `0` or an invalid value would do, i think.